src/core/execute.c

```diff
@@ -2022,14 +2022,13 @@ static int apply_mount_namespace(Unit *u, const ExecContext *context,
         char *tmp = NULL, *var = NULL;
         const char *root_dir = NULL;
         NameSpaceInfo ns_info = {
+                .force_protect_paths = true,
```

Comment:
The only user below uses `!force_protect`. It seems it'd be better to negate this variable. That'd be in line more with out places where we specify "allow ignore" as a positive flag.